### ISSUE TITLE
Functional connectivity methods are asymmetric

### ISSUE THREAD

**Issue by olivercliff** [Eden-Kramer-Lab/spectral_connectivity#22]
Hi all,

When using [this dataset](https://github.com/Eden-Kramer-Lab/spectral_connectivity/files/8843792/example_dataset.zip), we found that the following methods (that are named as functional on the main page) yield directed values:
`coherence_phase`
`phase_locking_value`
`phase_lag_index`
`weighted_phase_lag_index`

The dataset has brain regions on the first axis and time dimension on the second (so it should be transposed before inputting to the toolkit).

Any ideas?

**Comment by edeno** [comment]
You are correct about these measures being asymmetric. They are typically used for functional connectivity by taking the absolute value or magnitude of the signal. Since they can be used either way, I chose to leave them in their complex form to leave that choice to the user. I should be more explicit in the documentation about this.

**Comment by olivercliff** [comment]
Thanks for your response, Eric. It seems that only the `phase_locking_value` method is complex though (the rest are floats). Should the other ones also be asymmetric/complex?
